docs/HDRUK/2.2.1.structure.json

```diff
@@ -219,7 +219,7 @@
                                     "examples": null,
                                     "type": [
                                           "EmailAddress[{'anyOf': [{'format': 'email', 'type': 'string'}, {'type': 'null'}]}]",
-                                          "typing.List[typing.Optional[hdr_schemata.definitions.HDRUK.EmailAddress.EmailAddress]]",
+                                          "List",
```

Comment:
I think this is happening because the `types` field in the script that generates these docs is currently disabled: [here](https://github.com/HDRUK/schemata-2/blob/5942059c877642ce8bf17f6e2a7e9eef1e81bf9d/hdr_schemata/utils/create_markdown.py#L83).  

If I try to reenable it the docs generation script fails.  So for now I've left it disabled and run the script, which means some detail gets lost on fields that I haven't changed as part of the PR.  The docs update is in a commit on its own though so should be easy to revert if this is the wrong approach.